pages/docker_compose.md

```diff
@@ -61,6 +61,16 @@ If you are using Docker Machine on Mac or Windows, your Docker daemon has only l
 
 </div>
 
+
```

Comment:
So, for people not having sudo installed and who want to install a package... I'd rather put a generic title, something like "Installing packages".